docs/tutorials/01_neural_networks.ipynb

```diff
@@ -499,110 +776,212 @@
     "\n",
     "Further, the `CircuitQNN` allows to specify an `interpret` function to post-process the samples. This is expected to take a measured integer (from a bitstring) and map it to a new index, i.e. non-negative integer. In this case, the output shape needs to be provided and the probabilities are aggregated accordingly.\n",
     "\n",
-    "A `CircuitQNN` can be configured to return sparse as well as dense probability vectors. If no `interpret` function is used, the dimension of the probability vector scales exponentially with the number of qubits and a sparse recommendation is usually recommended. In case of an `interpret` function it depends on the expected outcome. If, for instance, an index is mapped to the parity of the corresponding bitstring, i.e., to 0 or 1, a dense output makes sense and the result will be a probability vector of length 2."
+    "A `CircuitQNN` can be configured to return sparse as well as dense probability vectors. If no `interpret` function is used, the dimension of the probability vector scales exponentially with the number of qubits and a sparse probability vector is usually recommended. In case of an `interpret` function it depends on the expected outcome. If, for instance, an index is mapped to the parity of the corresponding bitstring, i.e., to 0 or 1, a dense output makes sense and the result will be a probability vector of length 2."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 23,
-   "id": "posted-shoot",
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "from qiskit_machine_learning.neural_networks import CircuitQNN"
-   ]
-  },
+    "<b>Steps followed to define `CircuitQNN` after importing the required libraries:</b>\n",
```

Comment:
As for the steps, I would re-structure the section similarly to the previous ones. In this case, we try forward/backward passes for the 4 kinds of outputs, so I would define sub-sections such as: 4.1. define feature map, ansatz and combine in quantum circuit, 4.2. construct QNN, 4.3. Output: sparse integer probabilities, 4.4.  Output: dense integer probabilities, 4.5. Output: Samples, 4.6. Output: Parity Samples)

Reply:
Done